lib/selectors/peers.js

```diff
@@ -1,18 +1,16 @@
 import { chain } from 'underscore';
 import { createSelector } from 'reselect';
 
-// const getPeersFromState = ({ node: nodeState }) =>
-//   nodeState.peers ? nodeState.peers : [];
-
 const getPeers = (peers = []) => peers;
 
 export const getPeerCoordinates = async peers => {
   const getCoordsPromises = peers.map(async ({ addr, id }) => {
+    const key = SECRETS.ipstack || '';
     const ip = addr.split(':').slice(0, 1);
-    const response = await fetch(`http://ip-api.com/json/${ip}`);
+    const response = await fetch(`http://api.ipstack.com/${ip}?access_key=${key}&format=1`);
```

Comment:
Needs to be https

Reply:
Unfortunately doesn’t seem to be supported with a free account. I think if we really want this it’s something we’re going to have to build ourselves. there are databases you can download but too big to bundle for the front end so will need to develop our own service or allow backend hooks. Since this is primarily for display/demo purposes for now, I think https for peers map is a lower priority. Agree it probably shouldn’t be deployed on any live sites we run (then again, the biggest risk is basically just a Man in the middle attack for spoofing the locations right?)

Reply:
The biggest risk is that if bpanel is served over https, then the browser will block http requests, making the map not work. If we just want to get this working, then its good to go